### DISCUSSION TITLE
Admin email config

### DISCUSSION THREAD

**Discussion by Crashtest007** [elementor/elementor#13327] [Developers Community]
Have I missed something?

I inherited a website using elementor!

There is no ability to style the email received when a form is submitted to admin.

It is received in plain text!  and some fields are either not wanted or are so close together they are hard to read.

See below what I receive, does anyone have a solution figured yet?

I know you can change the message [all-fields] and replace with the shortcodes of each field required but that seems to be very long winded given the amount of fields I have in this form. Not to mention it look untidy when received in email.

Message: test hhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh
HTML Delivery Address Heading: 
Number: 9
Street: 9 ####ss Ct
Town: BA### BE###
Post Code: 45##
Divider: 

Qty Required: 0

Qty Required: 0

Qty Required: 0

Qty Required: 0

Qty Required: 0
bribie
Qty Required: 1

Qty Required: 0

Qty Required: 0

Qty Required: 0

Qty Required: 0
Divider: 
Maps $: 
Note: 
Acceptance: on

**Comment by Aviad-Herman** [comment]
Hey @Crashtest007,

You can style the email using HTML and CSS.

Check out this short tutorial:
https://elementor.com/help/sending-email-confirmations-with-elementor-forms/

Also make sure the "Send As" field is "HTML"
![image](https://user-images.githubusercontent.com/73067562/102764916-c1921680-4330-11eb-9442-0d86138e05f5.png)

**Reply by Crashtest007** [reply]
Thanks for the reply Aviad-Herman,
The HTML (send as ) appears to make no difference. But I will persist with entering the HTML in the message field after I have compiled it . Let's see the outcome.
One other thing that is odd  - When I remove the [all-fields] and enter the short codes I want sent - The resulting email parses the fields side by side with no line breaks.

Wayne

**Comment by Aviad-Herman** [comment]
I made a short tutorial for you:
https://www.loom.com/share/0f1857b8b6e14f7c9373e26826ece4cb

**Reply by Crashtest007** [reply]
Hi Aviad-Herman,

What a Brilliant and Simple way of doing this!  I never would have thought of it.

I guess I was trying to "Over Think" it.

Well done :)

Regards,
Wayne

**Reply by Crashtest007** [reply]
P.S. Thanks for the Great Video Tutorial.
